db/migrate/20231123233308_delete_duplicated_holidays.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class DeleteDuplicatedHolidays < ActiveRecord::Migration[7.0]
+  def up
+    execute <<~SQL
+      DELETE
+        FROM calendar_events ce
+      WHERE
+        ce.id IN (SELECT ce2.id FROM calendar_events ce2
+                  INNER JOIN users ON users.id = ce2.user_id
```

Comment:
Is it possible for a different user to create the event for someone else?

Reply:
No, Calendar events without post id are holidays only generated by scheduled job